app/src/main/dist/config/crate.yml

```diff
@@ -806,11 +806,11 @@
 # a query.
 
 # Specifies the limit for the query breaker.
-#node.breaker.query.limit: 60%
+#node.attr.breaker.query.limit: 60%
```

Comment:
I think this setting doesn't exist anymore. There is a `indices.breaker.query.limit` instead

Reply:
when did this change? I would like to put some notes in the release notes

Reply:
This has been changed a long time ago - back to 0.50.0: https://github.com/crate/crate/commit/927824ed96d747036b269c3eebc77975a540c163

Reply:
okay that's far enough back that we don't even have release notes on master so I can't even add breaking changes

I could go back and edit the 0.50 branch if you think it's necessary?